### PR TITLE
chore: add set operations

### PR DESCRIPTION
**Pull request by pgautier404**

### CODE REVIEW

File: momento/set_add_elements.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+	}
+
+	var ttl uint64
+	if ttl, err = prepareTTL(r, client.defaultTtl); err != nil {
+		return err
+	}
+
+	r.grpcRequest = &pb.XSetUnionRequest{
+		SetName:         []byte(r.SetName),
+		Elements:        momentoBytesListToPrimitiveByteList(r.Elements),
```
Comment (eaddingtonwhite) [inline momento/set_add_elements.go:54]: nit: but maybe this should be renamed to `momentoValuesToPrimitiveByteList`

File: momento/set_remove_elements.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	Elements  []Value
+
+	grpcRequest  *pb.XSetDifferenceRequest
+	grpcResponse *pb.XSetDifferenceResponse
+	response     SetRemoveElementsResponse
+}
+
+func (r *SetRemoveElementsRequest) cacheName() string { return r.CacheName }
+
+func (r *SetRemoveElementsRequest) requestName() string { return "SetAddElements" }
```
Comment (schwern) [inline momento/set_remove_elements.go:33]:
```suggestion
func (r *SetRemoveElementsRequest) requestName() string { return "SetRemoveElements" }
```

**Review by schwern — CHANGES_REQUESTED** [review]

**Comment by schwern** [comment]
@pgautier404 LGTM, just that one thing.

File: momento/set_remove_elements.go (hunk 2 above)
Comment (pgautier404) [reply momento/set_remove_elements.go:33]: Ugh, nice catch.

**Review by eaddingtonwhite — APPROVED** [review]

**Review by schwern — APPROVED** [review]

File: momento/set_add_elements.go (hunk 1 above)
Comment (pgautier404) [reply momento/set_add_elements.go:54]: Yep, good suggestion, thanks.

**Review by eaddingtonwhite — APPROVED** [review]

**Pull request merged**